contracts/metadata/metadata.cairo

```diff
@@ -0,0 +1,691 @@
+    }
+
+    namespace WonderNames {
+        const wonder_1 = 'Cathedral Of Agony';
+        const wonder_2 = 'Sanctum Of Purpose';
+        const wonder_3 = 'The Ancestral Willow';
+        const wonder_4 = 'The Crying Oak';
+        const wonder_5 = 'The Immortal Hot Spring';
+        const wonder_6 = 'Pantheon Of Chaos';
+        const wonder_7 = 'The Solemn Catacombs';
+        const wonder_8 = 'The Exalted Geyser';
+        const wonder_9 = 'The Devout Summit';
+        const wonder_10 = 'The Mother Grove';
+        const wonder_11 = 'Synagogue Of Collapse';
+        const wonder_12 = 'Sanctuary Of The Ancients';
+        const wonder_13 = 'The Weeping Willow';
+        const wonder_14 = 'The Exalted Maple';
+        const wonder_15 = 'Altar Of The Void';
+        const wonder_16 = 'The Pure Stone';
+        const wonder_17 = 'The Celestial Vertex';
+        const wonder_18 = 'The Eternal Orchard';
+        const wonder_19 = 'The Amaranthine Rock';
+        const wonder_20 = 'The Pearl Summit';
+        const wonder_21 = 'Mosque Of Mercy';
+        const wonder_22 = 'The Mirror Grotto';
+        const wonder_23 = 'The Glowing Geyser';
+        const wonder_24 = 'Altar Of Perfection';
+        const wonder_25 = 'The Cerulean Chamber';
+        const wonder_26 = 'The Mythic Trees';
+        const wonder_27 = 'The Perpetual Ridge';
+        const wonder_28 = 'The Fading Yew';
+        const wonder_29 = 'The Origin Oasis';
+        const wonder_30 = 'The Sanctified Fjord';
+        const wonder_31 = 'The Pale Pillar';
+        const wonder_32 = 'Sanctum Of The Oracle';
+        const wonder_33 = 'The Ethereal Isle';
+        const wonder_34 = 'The Omen Graves';
+        const wonder_35 = 'The Pale Vertex';
+        const wonder_36 = 'The Glowing Pinnacle';
+        const wonder_37 = 'The Azure Lake';
+        const wonder_38 = 'The Argent Catacombs';
+        const wonder_39 = 'The Dark Mountain';
+        const wonder_40 = 'Sky Mast';
+        const wonder_41 = 'Infinity Spire';
+        const wonder_42 = 'The Exalted Basin';
+        const wonder_43 = 'The Ancestral Trees';
+        const wonder_44 = 'The Perpetual Fjord';
+        const wonder_45 = 'The Ancient Lagoon';
+        const wonder_46 = 'The Pearl River';
+        const wonder_47 = 'The Cerulean Reliquary';
+        const wonder_48 = 'Altar Of Divine Will';
+        const wonder_49 = 'Pagoda Of Fortune';
+        const wonder_50 = 'The Oracle Pool';
+    }
+}
+
+namespace Uri {
+    // @notice build uri array from stored realm data
+    // @implicit syscall_ptr
+    // @implicit bitwise_ptr
+    // @implicit range_check_ptr
+    // @param realm_id: id of the realm
+    // @param realm_name: encoded string of the realm name
+    // @param realm_data: unpacked data for realm
+    // @param realm_type: type of realm (Realm or S_Realm)
+    func build{
+        syscall_ptr: felt*,
+        pedersen_ptr: HashBuiltin*,
+        range_check_ptr,
+        bitwise_ptr: BitwiseBuiltin*,
```

Comment:
Only `range_check_ptr` is needed here. You can remove `syscall_ptr` from the rest of the functions in this module as well, `range_check_ptr` is the only one necessary.

Reply:
This wasn't resolved @starknetdev 🙁